src/shared/remoteSession.ts

```diff
@@ -173,13 +180,47 @@ export async function handleMissingTool(tools: Err<MissingTool[]>) {
 export async function getDeniedSsmActions(client: IamClient, roleArn: string): Promise<IAM.EvaluationResult[]> {
     const deniedActions = await client.getDeniedActions({
         PolicySourceArn: roleArn,
-        ActionNames: [
-            'ssmmessages:CreateControlChannel',
-            'ssmmessages:CreateDataChannel',
-            'ssmmessages:OpenControlChannel',
-            'ssmmessages:OpenDataChannel',
-        ],
+        ActionNames: minimumSsmActions,
     })
 
     return deniedActions
 }
+
+export async function promptToAddPolicies(client: IamClient, roleArn: string): Promise<boolean> {
+    const promptText = `In order to connect, toolkit will add actions ${getFormattedSsmActions()} to role ${roleArn}`
```

Comment:
The most verbose variable should usually go at the end of a message, else it crowds out parts that a human actually needs to read.

```suggestion
    const promptText = `${getIdeProperties().company} Toolkit will add required actions to role ${roleArn}:  ${getFormattedSsmActions()}`
```